test/functional/compute_attribute_test.rb

```diff
@@ -574,23 +574,23 @@
                                      "id" => 2,
                                      "compute_resource_id" => 1,
                                      "compute_resource_name" => "bla",
-                                     "provider_friendly_name" => "oVirt",
+                                     "provider_friendly_name" => "Libvirt",
                                      "compute_profile_id" => 1,
                                      "compute_profile_name" => "profile2",
                                      "vm_attrs" => {"interfaces_attributes" => { "1" => {"compute_name"=>"eth0"}}}
```

Comment:
For `libvirt` interface attributes should have a different key:
```suggestion
                                     "vm_attrs" => {"nics_attributes" => { "1" => {"compute_name"=>"eth0"}}}
```

Reply:
Oooh, I was under the impression that we had unified the parameters, so it's a problem elsewhere. Thanks; I will update the PR.